### PR TITLE
Add note about Google Maps coordinates in Safehomes docs

### PR DESCRIPTION
**Pull request by avsaase**

### CODE REVIEW

File: docs/Safehomes.md [hunk 1]
```diff
@@ -53,7 +53,9 @@ Parameters:
   * `<lat>` - Latitude (WGS84), in degrees * 1E7 (for example 123456789 means 12.3456789).
   * `<lon>` - Longitude.
 
-Safehomes are saved along with your regular settings and will appear in `diff` and `dump` output.  Use `save` to save any changes, as with other settings.
+Note that coordinates from Google Maps only have six decimals, so you need to remove the decimal dot and add a zero to the end to set the correct safehome location.
```
Comment (digitalentity) [inline docs/Safehomes.md]: I think an example would be nice here.

Comment (avsaase) [reply docs/Safehomes.md]: Good point. Have a location in mind?

Comment (avsaase) [reply docs/Safehomes.md]: Used the same as the existing example.

**Comment by avsaase** [comment]
Btw, why do we have 7 decimals in the first place? This corresponds to 1.11 cm accuracy, completely unnecessary. 5 decimals would be sufficient IMO.

**Review by digitalentity — APPROVED** [review]

**Pull request merged**